content/software/ide-v1/tutorials/getting-started/cores/arduino-avr/arduino-avr.md

```diff
@@ -19,11 +19,11 @@ You can download the editor from [our software page](https://www.arduino.cc/en/s
 ### Boards using the AVR core
 
 - Arduino UNO R3
```

Comment:
```suggestion
- Arduino UNO R3
- Arduino UNO R3 SMD
```

Reply:
@karlsoderby Do we add the R3 SMD to this list?

Reply:
Yes I think it is correct to add the SMD version as well!